utils/message_scheduler.py

```diff
@@ -73,7 +73,7 @@ async def send_daily_question_and_update_stats(force_update_stats: bool = True,
             await send_leaderboard_winners(server, "last_week")
 
     if daily_reset:
-        embed = await daily_question_embed()
+        embed = await get_daily_question()
```

Comment:
why did you rename the function from daily_question_embed. Keep it as daily_question_embed.

Reply:
actually, look at my feedback in utils/questions. Don't rename get_daily_question, but make it return data rather than an embed. and create a new daily_question_embed that uses get_daily_question, uses question_embed, and returns that embed.